Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/CrowdStrikeFalcon_test.py

```diff
@@ -5495,6 +5499,779 @@ def test_run_batch_write_cmd_timeout_argument(mocker, timeout, expected_timeout)
     assert request_mock.call_args[1].get('params').get('timeout') == expected_timeout
 
 
+<< << << < HEAD
+# New Tests
+
+
+def assert_command_results(command_results_to_assert: CommandResults, expected_outputs: list | dict,
+                           expected_outputs_key_field: str | list[str],
+                           expected_outputs_prefix: str):
+    """This function is used to assert the command results object returned from running command using mocked data.
+    It checks the three important fields, which are:
+    1. outputs
+    2. outputs_key_field
+    3. outputs_prefix
+
+    Args:
+        command_results_to_check (CommandResults): The command results object to assert.
+        expected_outputs (list | dict): The expected outputs object.
+        expected_outputs_key_field (str | list[str]): The expected outputs key field object.
+        expected_outputs_prefix (str): The expected outputs prefix object.
+    """
+    assert command_results_to_assert.outputs == expected_outputs
+    assert command_results_to_assert.outputs_key_field == expected_outputs_key_field
+    assert command_results_to_assert. outputs_prefix == expected_outputs_prefix
+
+
+def test_format_time_to_custom_date_format(mocker: MockerFixture):
+    ...
+
+
+class TestCSFalconCSPMListPolicyDetialsCommand:
+
+    def test_http_request_with_status_code_400_500_207(self, mocker: MockerFixture):
+        """
+        Validate that the http request of the command cs-falcon-cspm-list-policy-details
+        accepts the status codes 500, 400, and 207, since we deal with them manually.
```

Comment:
do you run each? or what exactly? please describe all unit tests using `given/when/then` as usual

Reply:
The http_request function accepts an argument, status_code, that lets us define which status codes we don't want to raise an error for, so we can deal with them manually, therefore, we assert that the http_request is called with the correct status codes